### ISSUE TITLE
Change structure of Contributing Guide

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/light-client#186]
## Description

Change the structure of the contributing.md according to the development process a developer is following

Proposal

```
- [Creating an Issue](#creating-an-issue)
  * [For Feature Requests](#for-feature-requests)
  * [For Bugs](#for-bugs)
- [Creating a Pull Request](#creating-a-pull-request)
  * [Prepare](#prepare)
    + [Installation](#installation)
    + [Test set-up](#test-set-up)
  * [Implement](#implement)
    + [Coding Style](#coding-style)
    + [Writing a test](#writing-a-test)
    + [Documentation](#documentation)
    + [Commiting Rules](#commiting-rules)
    + [Opening a pull request](#opening-a-pull-request)
    + [Integrating Pull Requests](#integrating-pull-requests)
```

## Acceptance criteria
- 

## Tasks
- [ ]

**Comment by nephix** [comment]
I'm unassigning myself, since I'll be unavailable for quite some time.

**Comment by nephix** [comment]
Has been merged, follow up ticket is #758
